pkg/services/dashboardimport/service/service_test.go

```diff
@@ -98,7 +98,6 @@ func TestImportDashboardService(t *testing.T) {
 		require.Equal(t, "prom", panel.Get("datasource").MustString())
 
 		require.True(t, importLibraryPanelsForDashboard)
-		require.True(t, connectLibraryPanelsForDashboardCalled)
```

Comment:
Connections are now added in a different flow: https://github.com/grafana/grafana/pull/106734